docs/guide/laravel.md

```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
 Controller-Method: ArticleController@getList
 Model-Method: Article::getList
 ```
+  
+## Performance Tuning SQL Queries
+
+SQL tuning is the process of improving SQL queries to accelerate your servers performance.
+It's general purpose is to reduce the amount of time it takes a user to 
+receive a result after issuing a query, and to reduce the amount of 
+resources used to process a query. 
+
+Big SQL queries can suffer performance issues. 
+Performance degradation often occurs if the database is not being properly 
+maintained or if queries can be rewritten more efficiently.
+
+Following Query optimization tips for better performance
+
+- **Apply Index On Necessary Columns:**
+
+Table indexes in databases help retrieve information faster and more efficiently.
+An index creates a unique data column without overlapping each other. 
+It improves the speed of data retrieval.
+
+- **Avoid Query In A Loop:**
+ 
+ If sometimes we need to execute the same query multiple times like if 
+ we need to insert 10 records in any table at that time don’t use 
+ insert query in the loop instead of using bulk insert.
+ 
+ We shouldn't call Eloquent relationships method in loop. 
+ First convert model object in to collection and then pass in to loop.
+ 
+  
+ **Example:**
+ ```php
+ 
+ 
+ public function category()
+     {
```

Comment:
@we-vikram-wri231 fix intending https://drive.google.com/file/d/1rdxhNuiXF67HcJQBmgqcf3WK1IipEpsS/view

Reply:
@themodernpk Sir,
Fixed intending